data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+				goto choose
+			label whatevercost
+			`	He seems a bit wary of your attitude, "Please, whatever you do, try to avoid needless bloodshed, these pirates are not our enemies. I would hardly recommend bribing those that can be. That said, I expect you to do what you must in order to bring back all 142 tons of materials."`
+			label finalmention
+			`	"You can either fit all of the cargo on a Shield Beetle, or bring them back in multiple trips, by storing them in an outfitter or on your other ships. And remember to look for it north-east of human space."`
+			action
+				clear "unfettered: worthless question"
+				clear "unfettered: why not hire another question"
+			branch nvm
+				has "flagship model: Shield Beetle"
+			`	"You must know of the current food shortages we are facing. Far from me to suggest you should profit from it, but I know one can easily make a lot of money moving cargo around in Aphids with the tribute and aid jobs, especially if they do not risk being attack by us, ha!"`
+				to display
+					has "unfettered: hunger"
+					credits < 32000000
+			`	"I would advise you to lead this mission with a Shield Beetle, like a true warrior. You deserve it, and it could prove useful."`
+			label nvm
+			`	"It would also probably be best to limit yourself to say 2 heavy warships so you don't attract too much attention."`
+			# no need to accept because this is a job; it was already accepted.
+	on enter "Misam"
+		conversation
+			`As you enter the system, you notice pirates, presenting themselves as part of the Cerberus operations, in orbit around the planet. They could have the cargo you are looking for.`
+			choice
+				`	"I demand you return the materials to me."`
+				`	(I have no need to talk with these scums. I shall take.)`
+					goto over
+			`	The first mate of the <npc> responds to your hail, "You talking about the cargo we just relieved from that scum posing as a merchant? That's ours, mate, always has been. You must have suffered a serious blow to the head to think otherwise. In fact we were the ones that these were stolen from before we could bring it to port! Well, after we stole them from an actual merchant, but that's irrelevant."`
+			choice
+				`	"What do you intend to do with this cargo?"`
+					goto intention
+				`	"If you do not give it to me, I will open fire."`
+			`	"If I laid down at every threat I received, the Masters would have replaced me long ago. Not gonna lie, I don't mind bashing you down a bit."`
+				goto over
+			label intention
+			`	"That's none of your business, but if it can help you get away... My Masters are coming soon to retrieve the cargo quite a few systems away from here, and if it ain't there I'll be the one they 'retrieve' instead. So forget about it!" He says, before abruptly closing the communication.`
```

Comment:
```suggestion
			`	"That's none of your business, but if it can help you get away... my Masters are coming soon to retrieve the cargo quite a few systems away from here, and if it ain't there I'll be the one they 'retrieve' instead. So forget about it!" He says, before abruptly closing the communication.`
```